### PR TITLE
India minute

### PR DESCRIPTION
**Pull request by lintal**

### CODE REVIEW

File: cloudformationTemplate.json [hunk 1]
```diff
@@ -0,0 +1,64 @@
+{
+  "AWSTemplateFormatVersion": "2010-09-09",
+  "Transform": "AWS::Serverless-2016-10-31",
+  "Description": 
+    "Legacy 'WS Media API' (formerlly known as 'PodcastAPI')",
```
Comment (jcable) [inline cloudformationTemplate.json]: formerly

File: index.js [hunk 2]
```diff
@@ -346,6 +350,20 @@ exports.handler = async (event, context, callback) => {
             }
 
             await Promise.all(promises).then(() => {
+              // sort programmes on priority
+              const priorityMap = {};
+              if (programmes[0].priority) {
+                for (let p=0 ; p < programmes.length ; p++) {
+                  priorityMap[parseInt(programmes[p].priority)] = p;
```
Comment (jcable) [inline index.js]: Does this require that if any programme has a priority they all must?

Should we have:

                  programmes[p].priority && priorityMap[parseInt(programmes[p].priority)] = p;

**Comment by jcable** [comment]
Test needed for priority mechanism. This is also make sure we have an example of the products.json with priorities in it.

**Comment by jcable** [comment]
Add description of priority mechanism to README.

File: cloudformationTemplate.json (hunk 1 above)
Comment (lintal) [reply cloudformationTemplate.json]: #f67a059

File: index.js (hunk 2 above)
Comment (lintal) [reply index.js]: Yes I currently make an assumption that, if the first item in the `media` property in `products.json` for that product, then all media items should have this property too.

Just trying to figure out what business-rule you are trying to achieve. Is it perhaps that I should put anything with a priority in order above anything else without priority? @jcable

**Comment by lintal** [comment]
> Add description of priority mechanism to README.

#b06699a

File: index.js (hunk 2 above)
Comment (lintal) [reply index.js]: Resolved in aa7198f

**Pull request merged**